test/e2e/run.sh

```diff
@@ -34,7 +34,12 @@ export SCP="scp $SSH_OPTS"
 export VM_SSH_USER=vagrant
 
 export nri_resmgr_src=${nri_resmgr_src:-"$SRC_DIR"}
-export nri_resmgr_cfg=${nri_resmgr_cfg:-"${SRC_DIR}/test/e2e/files/nri-resmgr-topology-aware.cfg"}
+
+if [ "$POLICY" == "topology-aware" ]; then
+    export nri_resmgr_cfg=${nri_resmgr_cfg:-"${SRC_DIR}/test/e2e/files/topology-aware/nri-resmgr-topology-aware.cfg"}
+elif [ "$POLICY" == "balloons" ]; then
+    export nri_resmgr_cfg=${nri_resmgr_cfg:-"${SRC_DIR}/test/e2e/files/balloons/nri-resmgr-balloons.cfg"}
+fi
```

Comment:
@jukkar These bits look like a fix not related to using kustomize, right ?

Reply:
It is indirectly related to the kustomize. To be exact, you can see below in this script that when the default configuration file is used, we are passing `nri_resmgr_cfg` variable to the kustomize. Since the way how policy configuration looks differs from policy to policy, I added those lines so that we use the correct default configuration files for each policy.